### PR TITLE
fmt(cstyle): do not validate src/config.h

### PR DESCRIPTION
**Pull request by marcoandredinis**
This file is auto-generated so it should not be validated when running `make
cstyle`.

You can find out more information here: 
https://github.com/openebs/openebs/issues/2009#issuecomment-427594728

### CODE REVIEW

**Review by utkarshmani1997 — APPROVED** [review]
LGTM

File: Makefile.in [hunk 1]
```diff
@@ -71,6 +71,7 @@ distdir:
 
 cstyle:
 	find $(top_srcdir) -name '*.[hc]' \
+                ! -wholename './src/config.h' \
```
Comment (vishnuitta) [inline Makefile.in]: can you please change `wholename` to `path`, being more portable?

**Review by vishnuitta — CHANGES_REQUESTED** [review]
changes as requested in comments please

File: Makefile.in (hunk 1 above)
Comment (marcoandredinis) [reply Makefile.in]: Sure!

**Review by vishnuitta — APPROVED** [review]
changes are good

**Pull request merged**